internal/datacoord/segment_manager.go

```diff
@@ -439,8 +442,7 @@ func (s *SegmentManager) SealAllSegments(ctx context.Context, collectionID Uniqu
 			ret = append(ret, id)
 			continue
 		}
-		if info.State == commonpb.SegmentState_Flushing ||
-			info.State == commonpb.SegmentState_Flushed {
+		if info.State != commonpb.SegmentState_Growing {
```

Comment:
Also might need to update here as well:
https://github.com/milvus-io/milvus/blob/master/internal/datacoord/segment_manager.go#L528